modules/platform/nuxeo-platform-usermanager-api/src/main/java/org/nuxeo/ecm/platform/usermanager/NuxeoPrincipalImpl.java

```diff
@@ -59,6 +59,11 @@
 
     protected UserConfig config = UserConfig.DEFAULT;
 
+    /**
+     * @since 11.4
+     */
+    protected boolean isPartial = false;
+
```

Comment:
Can you move it at the end of field declarations? No need to init it to `false` it is already the default value.